README.md

```diff
@@ -169,3 +172,17 @@ For `dsub` server documentation, see [servers/dsub](servers/dsub/README.md).
 
 ## Job Manager `cromwell` Server
 For `cromwell` server documentation, see [servers/cromwell](servers/cromwell/README.md).
+
+## Build docker images and releases
+From v0.2.0, Job Manager starts to release stock docker images on [DockerHub](https://hub.docker.com/u/databiosphere)
+
+- To build the `job-manager-ui` image with `$TAG` from the root of this Github repository:
+    ```
+    cd ui && docker build -t job-manager-ui:$TAG . -f Dockerfile
+    ```
+    
+- To build the `job-manager-api` image with `$TAG` from the root of this Github repository:
+    ```
+    docker build -t job-manager-api:v0.2.0 . -f servers/cromwell/Dockerfile
```

Comment:
What is `api`? Is it *both* cromwell and dsub shims bundled together? Would it sensible or completely insane to split them into two different images?

Reply:
This is a good catch! Actually, it was my fault that I totally forgot the dsub image. In fact, dsub and Cromwell shim layers have their own separate Dockerfiles(and in turn separate images), so we **should** have 
- job-manager-api-cromwell:v0.2.0
- job-manager-api-dsub:v0.2.0

separately.  I need to make changes throughout this PR for this. The only concern of mine is that thus we will have 3 docker images/docker repos just for Job Manager, do you think this would be annoying to users? @cjllanwarne

Reply:
Fixed now.